### DISCUSSION TITLE
SSR media query

### DISCUSSION THREAD

**Discussion by HellYeahOmg** [jpudysz/react-native-unistyles#60] [Q&A] [answered]
I'm using a monorepo with solito expo + next.js (pages router for ssr).

Config:
`
export const breakpoints = {
  xs: 0,
  sm: 576,
  md: 768,
  lg: 992,
  xl: 1200,
  superLarge: 2000,
  tvLike: 4000,
}
`
Component: 
`      <View style={styles.linksContainer}>
          ...
      </View>
`
Styles:
`linksContainer: {
    display: {
      xs: 'none',
      xl: 'flex',
    },
    flexDirection: 'row',
  },`

On the first render on the desktop (xl size) the linksContainer is hidden, even though it shouldn't be. Moreover, if I resize the window to a smaller, and then to a larger screen size (xl+), the media query seems to start working.

I also get a hydration error:
<img width="1213" alt="image" src="https://github.com/jpudysz/react-native-unistyles/assets/29250449/3ae944a1-9b84-4a8b-bf21-471935cfca7c">

**Comment by jpudysz** [comment]
That's definitely an issue,please create one

**Reply by jpudysz** [reply, marked as answer]
Resolved, updated demo https://github.com/jpudysz/react-native-unistyles-ssr-exmaple
